src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/Public.Data/PreviewTokenViewModel.cs

```diff
@@ -17,11 +17,12 @@ public record PreviewTokenViewModel
 
     public required string CreatedByEmail { get; set; }
 
-    public required DateTimeOffset Created { get; set; }
+    public required DateTimeOffset? Activates { get; set; }
```

Comment:
Highlighting a change to `Activates` which has crept into this PR. Not sure if this is intentional or not.